.travis.yml

```diff
@@ -62,6 +61,7 @@ jobs:
     - env: TOXENV=py34
     - env: TOXENV=py36
     - env: TOXENV=linting PYTEST_NO_COVERAGE=1
+    - env: TOXENV=docs PYTEST_NO_COVERAGE=1
```

Comment:
We could also use "TOXENV=linting docs" here (single job).

Reply:
Ahh good idea, will do.

Reply:
Done 👍